src/main/scala/no/ndla/conceptapi/service/ReadService.scala

```diff
@@ -7,34 +7,42 @@
 
 package no.ndla.conceptapi.service
 
-import no.ndla.conceptapi.repository.ConceptRepository
+import no.ndla.conceptapi.repository.{DraftConceptRepository, PublishedConceptRepository}
 import no.ndla.conceptapi.model.api
 import no.ndla.conceptapi.model.api.NotFoundException
 import no.ndla.conceptapi.model.domain.Language
 
 import scala.util.{Failure, Success, Try}
 
 trait ReadService {
-  this: ConceptRepository with ConceptRepository with ConverterService =>
+  this: DraftConceptRepository with PublishedConceptRepository with ConverterService =>
   val readService: ReadService
 
   class ReadService {
 
     def conceptWithId(id: Long, language: String, fallback: Boolean): Try[api.Concept] =
-      conceptRepository.withId(id) match {
+      draftConceptRepository.withId(id) match {
         case Some(concept) =>
           converterService.toApiConcept(concept, language, fallback)
         case None =>
           Failure(NotFoundException(s"Concept with id $id was not found with language '$language' in database."))
       }
 
+    def publishedConceptWithId(id: Long, language: String, fallback: Boolean): Try[api.Concept] =
+      publishedConceptRepository.withId(id) match {
+        case Some(concept) =>
+          converterService.toApiConcept(concept, language, fallback)
+        case None =>
+          Failure(NotFoundException(s"A published Concept with id $id was not found with language '$language'."))
```

Comment:
Unødvendig å ha med published her.
```suggestion
          Failure(NotFoundException(s"A concept with id $id was not found with language '$language'."))
```